cmd/nodecmd/ssh.go

```diff
@@ -37,6 +41,9 @@ If no [cmd] is provided for the node, it will open ssh shell there.
 		RunE: sshNode,
 	}
 	cmd.Flags().BoolVar(&isParallel, "parallel", false, "run ssh command on all nodes in parallel")
+	cmd.Flags().BoolVar(&includeMonitor, "with-monitor", false, "include monitoring node for ssh cluster operations")
```

Comment:
the default is being changed here

Reply:
yes, I am aware of it.
monitor and avalanchego nodes are so different. I guess we should mention it in the release notes explicitly

Reply:
we should treat avalancego nodes as cluster and service nodes separatly from each other imho

Reply:
What do you think @sukantoraymond